### PR TITLE
chore: use spin_manifest_version

### PR DESCRIPTION
**Pull request by fibonacci1729**

### CODE REVIEW

**Review by itowlson — COMMENTED** [review]
I don't really feel qualified to approve this, sorry - I don't know if there are any compatibility considerations.  But the Spin-side changes look fine to me for what it's worth!

**Review by vdice — APPROVED** [review]
The site appeared to run fine from this branch locally.  We can double-check finickywhiskers.com after this is merged (will auto-deploy).

LGTM

**Pull request merged**
